cabot/cabotapp/views.py

```diff
@@ -704,7 +704,9 @@ class ServiceCreateView(LoginRequiredMixin, CreateView):
     model = Service
     form_class = ServiceForm
 
-    alert.update_alert_plugins()
+    def __init__(self, *args, **kwargs):
+        alert.update_alert_plugins()
+        super(ServiceCreateView, self).__init__(*args, **kwargs)
```

Comment:
Is this the code that the test in the separate commit tests? Can this and that test be in one commit, and in one PR if it's current django version compatible?